ctru-rs/src/services/gfx.rs

```diff
@@ -65,40 +65,53 @@ pub trait Screen: private::Sealed {
     ///
     /// [`Swap::swap_buffers`] must be called after this method for the configuration
     /// change to take effect.
+    #[doc(alias = "gfxSetScreenFormat")]
     fn set_framebuffer_format(&mut self, fmt: FramebufferFormat) {
         unsafe { ctru_sys::gfxSetScreenFormat(self.as_raw(), fmt.into()) }
     }
 }
 
-/// The top screen. Mutable access to this struct is required to write to the top
-/// screen's frame buffer. To enable 3D mode, it can be converted into a [`TopScreen3D`].
+/// The top LCD screen.
+///
+/// Mutable access to this struct is required to write to the top screen's frame buffer.
+///
+/// To enable 3D mode, it can be converted into a [`TopScreen3D`].
 pub struct TopScreen {
     left: TopScreenLeft,
     right: TopScreenRight,
 }
 
+/// The top LCD screen set in stereoscopic 3D mode.
+///
 /// A helper container for both sides of the top screen. Once the [`TopScreen`] is
 /// converted into this, 3D mode will be enabled until this struct is dropped.
 pub struct TopScreen3D<'screen> {
     screen: &'screen RefCell<TopScreen>,
 }
 
-/// A screen that can have its frame buffers swapped, if double buffering is enabled.
+/// Trait for screens that can have its frame buffers swapped, when double buffering is enabled.
 ///
 /// This trait applies to all [`Screen`]s that have swappable frame buffers.
 pub trait Swap: private::Sealed {
     /// Swaps the video buffers.
     ///
-    /// If double buffering is disabled, "swapping" the buffers has the side effect
-    /// of committing any configuration changes to the buffers (e.g. [`set_wide_mode`],
-    /// [`set_framebuffer_format`], [`set_double_buffering`]).
+    /// Even if double buffering is disabled, "swapping" the buffers has the side effect
+    /// of committing any configuration changes to the buffers (e.g. [`TopScreen::set_wide_mode()`],
+    /// [`Screen::set_framebuffer_format()`], [`Swap::set_double_buffering()`]), so it should still be used.
     ///
     /// This should be called once per frame at most.
-    ///
-    /// [`set_wide_mode`]: TopScreen::set_wide_mode
-    /// [`set_framebuffer_format`]: Screen::set_framebuffer_format
-    /// [`set_double_buffering`]: Screen::set_double_buffering
+    #[doc(alias = "gfxScreenSwapBuffers")]
     fn swap_buffers(&mut self);
+
+    /// Set whether to use double buffering.
+    ///
+    /// # Notes
+    ///
+    /// Double buffering is enabled by default.
+    /// [`Swap::swap_buffers`] must be called after this function for the configuration
+    /// change to take effect.
+    #[doc(alias = "gfxSetDoubleBuffering")]
+    fn set_double_buffering(&mut self, enabled: bool);
```

Comment:
Yeah, this makes more sense as a part of `Swap` 👍

Reply:
Indeed. I've made this change as part of the "documentation" PR since it was a bit dumb to link it in a different struct (also, I'm pretty sure it didn't work correctly with the 3D left-right sides before, but it should fine now).